posts/rails_memory.md

```diff
@@ -139,6 +163,8 @@ RUN apt-get install -y libjemalloc-dev libjemalloc2
 ENV LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so.2
 ```
 
+<!-- part 11 -->
+
 And, the memory was not high anymore. With the analysis team APIs called at around 15:00. It was totally OK.
```

Comment:
-> Finally the memory was not high anymore. It was totally OK when the analysis team called APIs (?) at around 15:00.

Reply:
https://github.com/HoangNguyen679/aphelios/pull/1/commits/7ff21e904a7b15b9bca33fd3f4db95a0c855e513 fixed